lib/superstore-sync.js

```diff
@@ -14,15 +14,19 @@ var store = {};
 var persist = true;
 
 // Watch for changes from other tabs
-window.addEventListener("storage", function(e) {
-  if (keys[e.key]) {
-    keys[e.key] = true;
-    store[e.key] = JSON.parse(e.newValue);
-  }
-});
+
 
 function Superstore(type) {
 	this.storage = window[type];
+  this.keys = {};
+  this.store = {};
+
+  window.addEventListener("storage", function(e) {
```

Comment:
can you put a todo in a comment to say something like

`// TODO: check the storageArea so that we only refresh the key when we need to`

[ recommend against writing the actual code in this pull request though because ought to have tests ]